distribution/docs/src/main/resources/_contents/_processing-framework/processing-framework-intro-contents.adoc

```diff
@@ -0,0 +1,172 @@
+[NOTE]
+====
+This code is experimental. While this interface is functional and tested, it may change or be removed in a future version of the library.
+====
+
+The *Asynchronous Processing Framework* is a way to run plugins asynchronously. Generally, plugins that take a significant amount of processing time and whose
+results are not immediately required are good candidates for being asynchronously processed. A *Processing Framework* can either be run on the local or
+remote system. Once the *Processing Framework* finishes processing incoming requests, it submits its results back to the Catalog as an update. The type of plugins that a *Processing Framework*
```

Comment:
Although our default `ProcessingFramework` submits `UpdateRequest`s, another `ProcessingFramework` may do whatever it wants once it "finishes processing incoming requests".

I suggest changing this sentence to "Once the *Processing Framework* finishes processing incoming requests, it may submit `(Create/Update/Delete)Request`s to the Catalog". This would match
- the `(Create/Update/Delete)Request` label in the `Processing Framework Architecture` diagram
- the description for `ProcessingFramework`